pypicloud/storage/s3.py

```diff
@@ -9,16 +9,24 @@
 
 import boto.s3
 from boto.s3.key import Key
+import boto.s3.connection
 from pyramid.httpexceptions import HTTPFound
 from pyramid.settings import asbool
 
 from .base import IStorage
 from pypicloud.models import Package
-from pypicloud.util import parse_filename, getdefaults
+from pypicloud.util import parse_filename, getdefaults, str2bool
```

Comment:
I think that the `asbool` function from `pyramid.settings` will do what you want